@here/harp-mapview/lib/Tile.ts

```diff
@@ -387,6 +388,18 @@ export class Tile implements CachedResource {
         return this.m_worldCenter;
     }
 
+    /**
+     * Gets the key to uniquely represent this tile (based on the [[tileKey]] and [[offset]]), note
+     * this key is only unique within the given [[DataSource]], to get a key which is unique across
+     * [[DataSource]]s see [[DataSourceCache.DataSourceCache]].
+     */
+    get uniqueKey(): number {
```

Comment:
Very minor, i think that uniqueKey will be _always_ used for any tile that's actually used, so i believe lazy optimization is not necessarily needed, maybe just create this `uniqueKey` in constructor ?

Reply:
agreed, and it won't change either, will move it.

Reply:
of course I was wrong, in order not to pollute the DataSource.getTile method, I didn't add the offset, and the offset is set manually later.

That means however, the uniqueKey needs to be updated when the offset is changed, I have fixed this.